### ISSUE TITLE
Looking up out of bounds points throws an error with size=1

### ISSUE THREAD

**Issue by ceesem** [seung-lab/cloud-volume#537]
I was inadvertently looking up supervoxels using points at the wrong resolution in a cloudvolume instance with bounded=False and fill_missing=True, but instead of returning 0 as expected, CloudVolume threw an an exception. This only happens for size=1. For other sizes I have tried, the data returns correctly as an array of 0s.

For example, using the Minnie dataset:

```python
import cloudvolume

cv = cloudvolume.CloudVolume(
    "graphene://https://minnie.microns-daf.com/segmentation/table/minnie3_v1",
    use_https=True,
    bounded=False,
    fill_missing=True,
)

cv.download_point([902, 9, 4], size=1)
```

Throws an error with the following traceback:

```
TypeError                                 Traceback (most recent call last)
/var/folders/3r/pxs024hs3wg_nrm2f_r3ntf80000gp/T/ipykernel_3306/2385703022.py in <module>
----> 1 cv.download_point([900, 4, 2], size=1)

~/opt/miniconda3/envs/analysis/lib/python3.8/site-packages/cloudvolume/frontends/graphene.py in download_point(self, pt, size, mip, parallel, coord_resolution, **kwargs)
    120       parallel = self.parallel
    121 
--> 122     return self.download(bbox, mip, parallel=parallel, **kwargs)
    123 
    124   def unique(

~/opt/miniconda3/envs/analysis/lib/python3.8/site-packages/cloudvolume/frontends/graphene.py in download(self, bbox, mip, parallel, segids, preserve_zeros, agglomerate, timestamp, stop_layer, renumber, coord_resolution)
    279       renumber = False # no point
    280 
--> 281     img = super(CloudVolumeGraphene, self).download(bbox, mip=mip, parallel=parallel, renumber=renumber)
    282     renumber_remap = None
    283     if renumber:

~/opt/miniconda3/envs/analysis/lib/python3.8/site-packages/cloudvolume/frontends/precomputed.py in download(self, bbox, mip, parallel, segids, preserve_zeros, agglomerate, timestamp, stop_layer, renumber, coord_resolution)
    627       parallel = self.parallel
    628 
--> 629     tup = self.image.download(
    630       bbox.astype(np.int64), mip, parallel=parallel, renumber=bool(renumber)
    631     )

~/opt/miniconda3/envs/analysis/lib/python3.8/site-packages/cloudvolume/datasource/precomputed/image/__init__.py in download(self, bbox, mip, parallel, location, retain, use_shared_memory, use_file, order, renumber)
    164       )
    165     else:
--> 166       return rx.download(
    167         bbox, mip,
    168         meta=self.meta,

~/opt/miniconda3/envs/analysis/lib/python3.8/site-packages/cloudvolume/datasource/precomputed/image/rx.py in download(requested_bbox, mip, meta, cache, lru, fill_missing, progress, parallel, location, retain, use_shared_memory, use_file, compress, order, green, secrets, renumber, background_color)
    142 
    143   if requested_bbox.volume() == 1:
--> 144     return download_single_voxel_unsharded(
    145       meta, cache, lru,
    146       requested_bbox, first(cloudpaths),

~/opt/miniconda3/envs/analysis/lib/python3.8/site-packages/cloudvolume/datasource/precomputed/image/rx.py in download_single_voxel_unsharded(meta, cache, lru, requested_bbox, filename, mip, fill_missing, compress_cache, secrets, renumber, background_color)
    239     cache_enabled = cache.enabled
    240 
--> 241   chunk_bbx = Bbox.from_filename(filename)
    242   label, _ = download_chunk(
    243     meta, cache, lru,

~/opt/miniconda3/envs/analysis/lib/python3.8/site-packages/cloudvolume/lib.py in from_filename(cls, filename, dtype)
    433   @classmethod
    434   def from_filename(cls, filename, dtype=int):
--> 435     match = FILENAME_RE.search(os.path.basename(filename))
    436 
    437     if match is None:

~/opt/miniconda3/envs/analysis/lib/python3.8/posixpath.py in basename(p)
    140 def basename(p):
    141     """Returns the final component of a pathname"""
--> 142     p = os.fspath(p)
    143     sep = _get_sep(p)
    144     i = p.rfind(sep) + 1

TypeError: expected str, bytes or os.PathLike object, not NoneType
```

**Comment by william-silversmith** [comment]
Sorry about that, I recently optimized single voxel access to use a fast path but I forgot to test out of bounds. Thanks for catching this!

**Comment by william-silversmith** [comment]
Fixed in 8.5.4!
